proto/cosmos/staking/v1beta1/query.proto

```diff
@@ -90,6 +90,9 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/cosmos/staking/v1beta1/params";
   }
+
+  // this could return current estimation of next validator set
```

Comment:
Yeah I like this.  Return what the new validator set would be _if_ the epoch ended right now

Reply:
Cool